rfcs/proposed/zip_view/README.md

```diff
@@ -0,0 +1,25 @@
+# zip_view Support for the oneDPL Range APIs with C++20
+
+## Introduction
+`std::ranges::zip_view` is powerful utility enables developers to combine two or more ranges into a single view,
+where each element is represented as a tuple containing corresponding elements from each input range.
+
+## Motivations
+`std::ranges::zip_view` is a convenient way to combine multiple ranges into a single view, where each element of
+the resulting range is a tuple containing one element from each of the input ranges. This can be particularly
+useful for iterating over multiple collections in parallel. `std::ranges::zip_view` is introduced starting with C++23,
+but there are many users who work with C++20 standard yet. So, oneDPL introduces `oneapi::dpl::ranges::zip_view`,
+which the same API and functionality as `std::ranges::zip_view`.
+
+
+### Key Requirements
```

Comment:
- yes.
- I think it should work (by the requirements)
- I think zip_view::iterator should be convertible to dpl::zip_iterator. I guess the unification (via one type) soesnt make sense due to dpl::zip_iterator requires only C++17, but zip_view and zip_view::iterator - C++20.
- I think any "underlying" views/ranges should/will work. There are no restrictions for the host execution, for a device - rather only device copyability. A general restriction (came from "range-based parallel algorithms paper") - random access "underlying" views/ranges and zip_view itself.
- From current perspective, it seems - no.

Reply:
Update: In case of device execution  - an additional requirement for "underlying" views: each underlying view should model a `view` (see C++ standard). Basically, it follows from device copyability requirement.

Reply:
Good; make sure these questions are discussed in the design document.